src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/List.cshtml

```diff
@@ -14,8 +14,13 @@
     }
 
     var formActionWithoutTypeName = Context.Request.PathBase + "/Admin/Contents/ContentItems";
+    int startIndex = (Model.Pager.Page - 1) * (Model.Pager.PageSize) + 1;
+    int endIndex = startIndex + Model.ContentItems.Count - 1;
 }
 
+<script asp-name="bootstrap-select" depends-on="admin" asp-src="~/OrchardCore.Resources/Scripts/bootstrap-select.min.js" debug-src="~/OrchardCore.Resources/Scripts/bootstrap-select.js" at="Foot"></script>
+<style asp-name="bootstrap-select" asp-src="~/OrchardCore.Resources/Styles/bootstrap-select.min.css" debug-src="~/OrchardCore.Resources/Styles/bootstrap-select.css"></style>
+
 <h1>@RenderTitleSegments(pageTitle)</h1>
 @using (Html.BeginForm("List", "Admin", FormMethod.Post, new { id = "items-form", autocomplete = "off" }))
```

Comment:
Can we use the html instead of `HtmlHelper`?

Reply:
Done